src/commands/logs.ts

```diff
@@ -42,27 +84,52 @@ function run(argv: Arguments) {
   return controller.appLogStream(appName, token).then((result) => {
     const logUrl = makeLogStreamUrl(argv, result);
     if (argv.debug) {
-      console.log(`Connecting to ${logUrl}.`);
+      console.error(`Connecting to ${logUrl}.`);
```

Comment:
so the practise is to write to console.log for stdout and console.error for stderr?

Reply:
Yes.